core/src/main/java/org/kohsuke/stapler/Function.java

```diff
@@ -94,9 +94,11 @@
     boolean bindAndInvokeAndServeResponse(Object node, RequestImpl req, ResponseImpl rsp, Object... headArgs) throws IllegalAccessException, InvocationTargetException, ServletException, IOException {
         try {
             Object r = bindAndInvoke(node, req, rsp, headArgs);
-            if (getReturnType()!=void.class)
-                renderResponse(req,rsp,node, r);
+            if (getReturnType() != void.class)
+                renderResponse(req, rsp, node, r);
```

Comment:
> these little changes improve the readability of the source code

Yes, very slightly, but they significantly _harm_ the readability of the PR. If you want to do a bulk reformat in a separate PR (and introduce a “diff wall” past which merging & cherry-picking becomes impractical), that is another matter.